main.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import torch
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+import CustomDataLoader
+import HighFreqVit
+
+def main():
+    customloader = CustomDataLoader.DataProcesser()
+    data = customloader.get_datasets(path = "./datasets")
+    batch_size = 50
```

Comment:
Batch_size seems to be too high - we need to upload more train dataset. Thus please decrease it a bit

```suggestion
batch_size = 16
```